docs: add readme for exchange def examples (#170)
energywebfoundation/ssi: apps/vc-api/docs/example-exchange-definitions/README.md

A short guide for exchange-definitions examples

> The flows in the different use cases (presentation exchange) can be represented with three variables, which are:

Reviewer: IMO, this is slightly clearer
```suggestion
The flows in the different use cases (presentation exchange) can be categorized into three types, which are:
```
